merchstore/templates/merchstore/product_list.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'merchstore/base.html' %}
+{% block content %}
+<main>
+	<h1>Merchstore</h1>
+	{% if user.is_authenticated %}
+		<a href="{% url 'merchstore:product_create' %}"><h4>Add your product here</h4></a>
+	{% endif %}
+	{% if user.is_authenticated and user.profile.products.all %}
```

Comment:
`user.profile.products.exists` instead of `user.profile.products.all`
https://docs.djangoproject.com/en/5.1/ref/models/querysets/#exists